pulp_container/app/tasks/sync_stages.py

```diff
@@ -63,7 +63,7 @@ async def run(self):
         total_blobs = []
 
         with ProgressReport(
-            message="Downloading tag list", code="downloading.tag_list", total=1
+            message="Downloading tag list", code="sync.downloading.tag_list", total=1
```

Comment:
This helps keep the stages separate in combined sync/publish tasks.  Container plugin doesn't publish but it would be good to follow the same pattern.

Reply:
there is one more `code` to update on line 94, can you update it as well?

Reply:
nvm, i am blind :D